.github/workflows/commit_checker.yaml

```
@@ -0,0 +1,28 @@
+name: 'Commit Message Check'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - main
+
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Commit Type
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(Refs|Fixes): #[0-9]+ '
```

```suggestion
          pattern: '^(Refs|Fixes) #YUPTOO-[0-9]+ - .+$'
```
Instead of `:` I would prefer to have `-` in between.